### PR TITLE
Add Charleston File

### PR DESCRIPTION
**Pull request by amypiazza00**
Please check out the list of spots to eat in Charleston.

### CODE REVIEW

File: amypiazza00-charleston.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Charleston SC
+
+### Restaurants
+
+- Husk
+- Gilligans
+- 167 Raw
+- DIG
```
Comment (diarmaidm) [inline amypiazza00-charleston.md]: @amypiazza00 is DIG an acronym?

**Comment by diarmaidm** [comment]
> Please check out the list of spots to eat in Charleston.

Looks good :)

File: amypiazza00-charleston.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Charleston SC
```
Comment (diarmaidm) [inline amypiazza00-charleston.md]: Is SC south Carolina (trying out commenting :) )

File: amypiazza00-charleston.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+### Restaurants
+
+- Husk
```
Comment (diarmaidm) [inline amypiazza00-charleston.md]: **Testing** the --markdown--

Comment (jetgo1000) [inline amypiazza00-charleston.md]: add Hall's Chophouse to restaurants :money_with_wings: :moneybag:

Comment (diarmaidm) [inline amypiazza00-charleston.md]: Sounds like a place I would like :meat_on_bone: ?

File: amypiazza00-charleston.md (hunk 1 above)
Comment (amypiazza00) [inline amypiazza00-charleston.md]: Yes.  Daniel Island Grill.

**Comment by NatashaE** [comment]
🚢 :shipit:

**Pull request merged**